<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>838</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="SideBAR" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>91</width>
     <height>841</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color: rgba(255, 255, 255, 0.5); /* 设置背景色为灰色 */
}
</string>
   </property>
   <layout class="QGridLayout" name="SideBAR_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="13" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0">
     <widget class="QToolButton" name="TodayList">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{   
	border-top: 3px outset transparent;           /* 上边框宽度3像素、突出显示、颜色为透明 */
	border-bottom: 7px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;
    min-width: 80px;                            /* 控件最小宽度 */
    min-height: 80px;
	background-color: rgba(255, 255, 255, 0.5);
	color: Black;
}

/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgb(205, 205, 205);
}

/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	border-left: 3px outset rgb(93, 95, 97);  /* 左边框宽度3像素、突出显示、颜色为深深灰 */
	background-color: rgb(246, 246, 246);	
}

QPushButton:default {
	border-left: 3px outset rgb(93, 95, 97);  /* 左边框宽度3像素、突出显示、颜色为深深灰 */
	background-color: rgb(246, 246, 246);	
}</string>
      </property>
      <property name="text">
       <string>今日待办</string>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
      <property name="arrowType">
       <enum>Qt::ArrowType::NoArrow</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QToolButton" name="planlist">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{   
	border-top: 3px outset transparent;           /* 上边框宽度3像素、突出显示、颜色为透明 */
	border-bottom: 7px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;
    min-width: 80px;                            /* 控件最小宽度 */
    min-height: 80px;
	background-color: rgba(255, 255, 255, 0.5);
	color: Black;
}

/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgb(205, 205, 205);
}

/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	border-left: 3px outset rgb(93, 95, 97);  /* 左边框宽度3像素、突出显示、颜色为深深灰 */
	background-color: rgb(246, 246, 246);	
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
      </property>
      <property name="text">
       <string>计划内</string>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>100</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="Title">
      <property name="minimumSize">
       <size>
        <width>86</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{   
	border-top: 3px outset transparent;           /* 上边框宽度3像素、突出显示、颜色为透明 */
	border-bottom: 7px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;
    min-width: 80px;                            /* 控件最小宽度 */
    min-height: 20px;
	background-color: rgba(255, 255, 255, 0.5);
	color: Black;
}</string>
      </property>
      <property name="text">
       <string>智记清单</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QToolButton" name="alllList">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{   
	border-top: 3px outset transparent;           /* 上边框宽度3像素、突出显示、颜色为透明 */
	border-bottom: 7px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;
    min-width: 80px;                            /* 控件最小宽度 */
    min-height: 80px;
	background-color: rgba(255, 255, 255, 0.5);
	color: Black;
}

/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgb(205, 205, 205);
}

/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	border-left: 3px outset rgb(93, 95, 97);  /* 左边框宽度3像素、突出显示、颜色为深深灰 */
	background-color: rgb(246, 246, 246);	
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
      </property>
      <property name="text">
       <string>全部</string>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QToolButton" name="imList">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{   
	border-top: 3px outset transparent;           /* 上边框宽度3像素、突出显示、颜色为透明 */
	border-bottom: 7px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;
    min-width: 80px;                            /* 控件最小宽度 */
    min-height: 80px;
	background-color: rgba(255, 255, 255, 0.5);
	color: Black;
}

/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgb(205, 205, 205);
}

/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	border-left: 3px outset rgb(93, 95, 97);  /* 左边框宽度3像素、突出显示、颜色为深深灰 */
	background-color: rgb(246, 246, 246);	
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
      </property>
      <property name="text">
       <string>重要</string>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="res.qrc">:/images/icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="margin">
       <number>14</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="nonum">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>150</y>
     <width>31</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="menutitle">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>26</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>计划内:</string>
   </property>
  </widget>
  <widget class="QLabel" name="date">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>ERROR</string>
   </property>
  </widget>
  <widget class="QWidget" name="EditWidget_2" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>670</y>
     <width>651</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="EditWidget">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="nowcheck">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
    color: rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>当前选中:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="CheckTab">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>43</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	border:none;
	border-radius:10px;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="del">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>43</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="new2">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
    color: rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>新建:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="TodDoEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>43</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	border:none;
	border-radius:10px;
}</string>
        </property>
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>在此键入待办内容</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
    color: rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>地点:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="localtion">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>43</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	border:none;
	border-radius:10px;
}</string>
        </property>
        <property name="placeholderText">
         <string>在此键入地点</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="isimportant">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox {
    color: rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>重要</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="clockcheck">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>提醒我:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dateTimeEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>170</width>
          <height>43</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>8</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2024</year>
          <month>11</month>
          <day>2</day>
         </datetime>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
    color: rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>截止:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>43</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="date">
         <date>
          <year>2024</year>
          <month>11</month>
          <day>2</day>
         </date>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>45</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>661</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QListWidget" name="listTODO">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget {
    background: transparent; /* 设置QListWidget背景透明 */
    border: none; /* 去除边框 */
}

QListWidget::item {
	border-radius:10px;
    min-height: 43px; /* 设置每个item最小高度为43px */
	background-color: rgba(255, 255, 255, 0.8);
	color:rgb(0, 0, 0);
}</string>
      </property>
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listlocation">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget {
    background: transparent; /* 设置QListWidget背景透明 */
    border: none; /* 去除边框 */
}

QListWidget::item {
	border-radius:10px;
    min-height: 43px; /* 设置每个item最小高度为43px */
	background-color: rgba(255, 255, 255, 0.8);
	color:rgba(0, 0, 0, 0.5);
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolButton" name="Copyright">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>810</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{
	background-color: rgba(255, 255, 255, 0);
	border: none;
	color: rgb(255, 255, 255);
	font-size:9px;
}
QToolButton:hover {
    text-decoration: underline;
}</string>
   </property>
   <property name="text">
    <string>This program complies with the GPL-2.0 license. © 2018-2024 Minecraft ZBD Studio</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
